credentials/bc-registries-dbc-cred-doc/DBC-cred-doc-v1

```diff
@@ -0,0 +1,720 @@
+- **BC Services Card digital identity** – the identity information about the individual is created when the individual first accesses the BC Registries system using their BC Services Card digital identity and authenticator. The BC Services Card digital identity is a high assurance credential, with the individual’s name matching the name on their foundational identity document
+- **Canada Revenue Agency** – the information is provided to the BC Registries system directly from a CRA system
+
+The source of each attribute is described in the [Attributes](#bookmark=id.d4k15yq1kvi3) section.
+
+#### Data Updates
+
+When a DBC credential is issued, its data reflects the current records in BC Registries’ system. Changes to those records will trigger a revocation (and often re-issuance) of the credential to ensure the credential continues to reflect the identity record; these cases are described in further detail in the section on Revocation, below.
+
+### Assurance
+
+In order to minimize risk to BC Registries, verifiers, companies, and the individuals who represent companies, in the initial production release MVP phase (minimum valuable product), the DBC credential will only be issued to individuals when BC Registries is confident in both:
+
+- Their identity
+- Their relationship to the business
+
+For this reason, in the MVP, the only individuals who may obtain a DBC credential from BC Registries are:
+
+- Sole Proprietorship Business Owners who accessed the new Business Registries system using the BC Services Card mobile application and self-registered their own proprietorship
+
+#### BC Services Card
+
+The BC Services Card digital identity is used by BC Registries for both its identity information and the security of its two-factor authentication. By design, the BC Services Card digital identity meets the BC Office of the CIO [_Identity Assurance Standard_](https://www2.gov.bc.ca/assets/gov/government/services-for-government-and-broader-public-sector/information-technology-services/standards-files/identity_assurance_standard.pdf) requirements for a High (3) Identity Assurance Level. The BC Services Card digital identity has been assessed twice, in 2019 and 2021, by the Treasury Board of Canada Secretariat (TBS) against the Public Sector Profile of the Pan-Canadian Trust Framework (PSP-PCTF). In both cases, it was assessed as being a Trusted Digital Identity Level 3 (high), as defined in Appendix A of the TBS [_Directive on Identity Management_](https://www.tbs-sct.canada.ca/pol/doc-eng.aspx?id=16577).
+
+### Revocation
+
+A DBC credential will be revoked in the following cases:
+
+1. The individual is issued a new DBC credential for a business for which they already have been issued one or more credentials. An individual may only have one active DBC credential for a given business at a time, and when they are issued a new one, all previously issued active DBC credentials will be revoked.
```

Comment:
I think numbered lists in markdown have "1. " before each item in the list. 
I think it's better to use a list than literal numbers because then you get the proper list item paragraph format.